0.wave1_data/2.cellprofiler_analysis/README.md

```diff
@@ -7,34 +7,23 @@ Each file corresponds to a dilation size (Figure 1):
 
 1) interstellar_wave1_dilate25 = Dilation structuring element disk with size 25 pixels (approx. half the average cell diameter)
 2) interstellar_wave1_dilate50 = Dilation structuring element disk with size 50 pixels (approx. equal to the average cell diameter)
-3) interstellar_wave1_dilate100 = Dilation structuring element disk with size 100 pixels (approx. double the average cell diameter)
+3) interstellar_wave1_dilate75 = Dilation structuring element disk with size 75 pixels (approx. double the average cell diameter). This was changed from 100 originally since it was using a lot of computational power.
 
 ![dilation_ex.png](figures/dilation_ex.png)
 
-> Figure 1. CellProfiler dilation sizes and respective objects. 
+> Figure 1. Example of CellProfiler dilation sizes and respective objects. 
 > This figure shows how the various sizes for dilation in the `DilateObjects` module impacts the nuclei in an image. 
 > The images for each dilation are split into the two objects, `Dilated Nuclei` which are the nuclei that are dilated and `Translocated Nuclei` which are the `Dilated Nuclei` with the original `Nuclei` subtracted from the object. 
 
-We decided to test with various dilation values since we do not know how far Gasdermin-D translocates.
-We will be able to assess the best dilate value during further analysis.
+We decided to test with various dilation values since we do not know how far Gasdermin-D translocates to assess if Gasdermin-D is being measured.
 
-## Step 1: Create LoadData csv with illumination functions included
+## Create LoadData csv with IC functions and extract features with `loaddata_and_analysis.ipynb` notebook
 
-To create the LoadData csv to use in the analysis pipeline, `pe2loaddata` uses the following structure:
-
-```sh
-pe2loaddata --index-directory <index-directory> path/to/config.yml path/to/output.csv --illum --illum-directory <illum-directory> --plate-id <plate-id> --illum-output output_with_illum.csv
-```
-
-To create the LoadData csv with illum, make sure to change the paths within the [create_loaddata_illum_csv.ipynb](create_loaddata_illum_csv.ipynb) to reflect your local machine paths before running the code below (e.g. a local path for my computer starts with `/home/jenna`).
+To create the sqlite files, we ran the pipeline in the CellProfiler GUI three independent times where each run had a different dilation size. 
```

Comment:
Were there pipeline files associated with this which were used? If so, consider including these for reproducibility.